src/ChangeHighlight.tsx

```diff
@@ -31,6 +31,7 @@ const ChangeHighlight: React.FC<Props> = ({
 }) => {
   const [shadowDOM, setShadowDOM] = useState([]);
   const isInitialMount = useRef(true);
+  const uniqueId = Math.floor(Math.random() * 100000)+1;
```

Comment:
move this inside the useEffect condition of the first mount, because this will be called everytime